pkg/golinters/gomnd.go

```diff
@@ -0,0 +1,21 @@
+package golinters
+
+import (
+	magic_numbers "github.com/tommy-muehle/go-mnd"
+	"golang.org/x/tools/go/analysis"
+
+	"github.com/golangci/golangci-lint/pkg/golinters/goanalysis"
+)
+
+func NewGomnd() *goanalysis.Linter {
```

Comment:
Nit: Similarly, I would name this `NewGoMND` to be consistent with standard Go naming conventions.